STORACHA.md

```diff
@@ -0,0 +1,173 @@
+# Storacha IPFS Setup and AES-256 Proof Encryption
+
+This guide explains how to set up IPFS uploads via Storacha for the dApp, how to generate and encrypt your Storacha proof with AES-256, and how to configure the required environment variables.
+
+The dApp server route `dapp/src/pages/api/w3up-delegation.js` expects:
+- `STORACHA_SING_PRIVATE_KEY` (required): your W3UP/Storacha ED25519 principal private key.
+- `STORACHA_PROOF` (required): either the plain proof string or a 64-hex key used to decrypt a local `encrypted_proof.bin`.
+
+If `STORACHA_PROOF` is exactly 64 characters, the app treats it as an AES key (hex) and decrypts a local `encrypted_proof.bin` using `dapp/src/utils/decryptAES256.ts`.
```

Comment:
You do some repetitions here. This is the same text as above almost. And the text bellow explaining the mode should be there instead I think.